typescript/infra/scripts/warp-routes/helm.ts

```diff
@@ -27,12 +29,16 @@ function getHelmReleaseName(route: string): string {
 }
 
 function getWarpRoutesHelmValues(configFilePath: string) {
+  // The path should be relative to the monorepo root
```

Comment:
Not a fan of my hack here so if anyone has suggestions on how to convert the `configFilePath` that's passed in as a CLI arg to be a path relative to the monorepo root, pls do share